diff -up parley-20.03.80/src/dashboard/barwidget.cpp.omv~ parley-20.03.80/src/dashboard/barwidget.cpp
--- parley-20.03.80/src/dashboard/barwidget.cpp.omv~	2020-03-23 23:06:12.012260663 +0100
+++ parley-20.03.80/src/dashboard/barwidget.cpp	2020-03-23 23:06:17.012228542 +0100
@@ -20,6 +20,7 @@
 #include <QDebug>
 #include <QWidget>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPen>
 
 // KDE
diff -up parley-20.03.80/src/dashboard/gradereferencewidget.cpp.omv~ parley-20.03.80/src/dashboard/gradereferencewidget.cpp
--- parley-20.03.80/src/dashboard/gradereferencewidget.cpp.omv~	2020-03-23 23:06:27.352162179 +0100
+++ parley-20.03.80/src/dashboard/gradereferencewidget.cpp	2020-03-23 23:06:30.736140472 +0100
@@ -20,6 +20,7 @@
 #include <QDebug>
 #include <QWidget>
 #include <QPainter>
+#include <QPainterPath>
 #include <QPen>
 
 // KDE
diff -up parley-20.03.80/src/editor/vocabularydelegate.cpp.omv~ parley-20.03.80/src/editor/vocabularydelegate.cpp
--- parley-20.03.80/src/editor/vocabularydelegate.cpp.omv~	2020-03-23 23:05:01.707714213 +0100
+++ parley-20.03.80/src/editor/vocabularydelegate.cpp	2020-03-23 23:05:09.098666354 +0100
@@ -33,6 +33,7 @@
 #include <QDBusInterface>
 #include <QKeyEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QToolTip>
 
 using namespace Editor;
diff -up parley-20.03.80/src/practice/summarybarwidget.cpp.omv~ parley-20.03.80/src/practice/summarybarwidget.cpp
--- parley-20.03.80/src/practice/summarybarwidget.cpp.omv~	2020-03-23 23:05:48.482412046 +0100
+++ parley-20.03.80/src/practice/summarybarwidget.cpp	2020-03-23 23:05:54.546372996 +0100
@@ -17,6 +17,7 @@
 #include <KLocalizedString>
 
 #include <QPainter>
+#include <QPainterPath>
 #include <QEvent>
 #include <QHelpEvent>
 #include <QHBoxLayout>
diff -up parley-20.03.80/src/utils.cpp.omv~ parley-20.03.80/src/utils.cpp
--- parley-20.03.80/src/utils.cpp.omv~	2020-03-23 23:05:22.266581190 +0100
+++ parley-20.03.80/src/utils.cpp	2020-03-23 23:05:29.140536790 +0100
@@ -17,6 +17,7 @@
 
 // Qt
 #include <QPainter>
+#include <QPainterPath>
 #include <QRect>
 
 // KEduVocDocument library
